docker-generator/src/main/java/org/ballerinax/docker/generator/models/DockerModel.java

```diff
@@ -184,11 +187,16 @@ public void setDockerHost(String dockerHost) {
         return externalFiles;
     }
 
-    public void setCopyFiles(Set<CopyFileModel> externalFiles) {
+    public void setCopyFiles(Set<CopyFileModel> externalFiles) throws DockerGenException {
         this.externalFiles = externalFiles;
-        externalFiles.stream()
-                .filter(CopyFileModel::isBallerinaConf)
-                .forEach(file -> addCommandArg(" --config " + file.getTarget()));
+        for (CopyFileModel externalFile : externalFiles) {
+            if (externalFile.isBallerinaConf()) {
```

Comment:
Shall we invert the condition to reduce the levels of nested conditions?

Eg - 
```java
if (!externalFile.isBallerinaConf()) {
    continue;
}
...

```